other/management/commands/push-notify.py

```diff
@@ -57,11 +58,19 @@ def handle(self, *args, **options):
 
                 # Get title
                 title = "InstiApp"
+                notification_type = "event"
```

Comment:
Don't assume defaults; the object could be a None, in which case none of the following might be triggered.